rust-bors.example.toml

```diff
@@ -19,3 +19,7 @@ approve = ["+approved"]
 try = ["+foo", "-bar"]
 try_succeed = ["+foobar", "+foo", "+baz"]
 try_failed = []
+
+# Labels that will block approval when present on a PR
+# (Optional)
+labels_blocking_approval = ["final-comment-period", "proposed-final-comment-period"]
```

Comment:
Should we add "needs-fcp" here, or is that a label we expect to keep even after FCP passed?

Reply:
Btw seems like `needs-fcp` indeed isn't removed after the FCP finishes (https://github.com/rust-lang/rust/pull/79278), so maybe we shouldn't mark it as blocking.